packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -239,6 +246,29 @@ export class VaBreadcrumbs {
     });
   }
 
+  /**
+   * Changes the route when uswds and useRoute props are true.
+   * Route will be set to the href value in the breadcrumb.
+   */
+  handleRouteChange(e: MouseEvent, href: string): void {
+    e.preventDefault();
+    window.history.pushState(null, '', href);
+    // create event to notify Router
+    const popStateEvent = new PopStateEvent('popstate');
+    // notify Router
+    window.dispatchEvent(popStateEvent);
+  }
+
+  /**
+   * Handle click event on breadcrumb when uswds is true
+   */
+  uswdsHandler(e: MouseEvent, href: string) {
+    if (this.useRouter) {
+      this.handleRouteChange(e, href);
```

Comment:
What about situations where teams are needing both `Link` and `a` tags like here?

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/combined-debt-portal/debt-letters/containers/DebtDetails.jsx#L57-L67

Reply:
@jamigibbs  This situation would definitely be hard to handle with the current approach without, say, adding complexity to the breadcrumbList prop, ie for each breadcrumb another property like `isLink`.